ui/src/app/features/user-list/user-list-facade.ts

```diff
@@ -82,7 +82,7 @@ export class UserListFacade implements IFacade {
       deleteUser({
         userId: deletion.userToDelete.userId,
         deleterUserId: deletion.deleterUserId,
-        newOwner,
+        newOwner: 'owner',
```

Comment:
`newOwner: 'deleter'` is never used anymore?

Reply:
@isobolev, yes, now it will be always a global owner

Reply:
then we should remove the replacer parameter at all on BE. For now, let's hardcode owner in the endpoint call itself, and remove passing newOwner prop.